### PR TITLE
Unique id for toggle

### PR DESCRIPTION
**Pull request by theianjones**
# Changes
- Add unique id for htmlFor

## notes:
If there were two toggles on the page the left radio input would toggle the first `Toggle`

### CODE REVIEW

File: src/components/Toggle/index.js [hunk 1]
```diff
@@ -34,16 +34,14 @@ class Toggle extends Component {
 
     return (
       <div className='w5 relative flex items-center br-pill ba b--light-navy bg-dark-navy'>
-        <label className={labelClasses} htmlFor='left'>
+        <label className={labelClasses} htmlFor={`left-${leftOption}`}>
```
Comment (tayiorbeii) [inline src/components/Toggle/index.js]: Do we need to do something to handle any spaces that may be in the `leftOption` ?

`${leftOption.replace(' ', '-')}`

**Review by tayiorbeii — APPROVED** [review]

File: src/components/Toggle/index.js (hunk 1 above)
Comment (theianjones) [reply src/components/Toggle/index.js]: That's a good point, I'll add that ^

**Review by tayiorbeii — APPROVED** [review]
Any reason to replace with a regex instead of just a space?

**Review by trevordmiller — APPROVED** [review]

**Comment by theianjones** [comment]
If there is more than one space, only the 1st space will be filled with a hyphen, regex replaces all of them. I can't think of any reasons not to use regex in this case. What do you think @tayiorbeii?
```
> "a string with spaces".replace(/\s/g, "+")
'a+string+with+spaces'
> "a string with spaces".replace(" ", "-")
'a-string with spaces'
```

**Comment by tayiorbeii** [comment]
@ijones16 Good call! Regex works 👍

**Comment by trevordmiller** [comment]
Hey @ijones16, the `3.2.1` git tag never got pushed so a new npm version wont get published; did you run `yarn bump` instead of bumping the `package.json` version manully? If so, you must have had a dirty git state and the `yarn bump` script threw an error. You'll need to run `git push --tags` manually. In the future, make sure to commit, stage, push, then `yarn bump` and then the `package.json` version and git tags will match and a new npm version will get published.

**Comment by theianjones** [comment]
Sorry about that, I ran `yarn bump` but I think my git state was dirty. 

I just ran `git push --tags`

**Comment by trevordmiller** [comment]
@ijones16 Oh no problem, just wanted to make sure your package got pushed ok. You can see that it has been released correctly now so you should be good to `yarn upgrade egghead-ui` and get `3.2.1`: https://www.npmjs.com/package/egghead-ui

**Pull request merged**